python_modules/dagit/dagit/app.py

```diff
@@ -69,22 +69,27 @@ def info_view():
     )
 
 
-def notebook_view(request_args):
+def notebook_view(context, request_args):
```

Comment:
alex has been touching this path recently while moving dagit off of flask, he may have some thoughts here

Reply:
just make sure to add or edit a `dagit` python  test so it gets handled in the migration 

you can see current progress at `dagit.starlette`